randovania/data/json_data/prime3.txt

```diff
@@ -719,23 +797,28 @@ Asset id: 10553270987934706862
 > Dock to Falls of Fire; Heals? False
   * Normal Door to Falls of Fire/Dock to G.F.S. Theseus
 
-> Dock to Crash Site; Heals? False
-  * Normal Door to Crash Site/Dock to G.F.S. Theseus
+> Dock to Crash Site (Door); Heals? False
+  * Normal Door to Crash Site/Dock to G.F.S. Theseus (Door)
 
-> Dock to Crash Site; Heals? False
-  * Normal Door to Crash Site/Dock to G.F.S. Theseus
+> Dock to Crash Site (Tunnel); Heals? False
+  * Morph Ball Door to Crash Site/Dock to G.F.S. Theseus (Tunnel)
 
 ----------------
 Crash Site
 Asset id: 15814834557214288722
-> Dock to G.F.S. Theseus; Heals? False
-  * Normal Door to G.F.S. Theseus/Dock to Crash Site
+> Dock to G.F.S. Theseus (Door); Heals? False
+  * Normal Door to G.F.S. Theseus/Dock to Crash Site (Door)
 
 > Dock to Gateway; Heals? False
   * Normal Door to Gateway/Dock to Crash Site
```

Comment:
This should have a "Grapple Swing  or SA” requirement for crossing the gap

Reply:
Readjusted and fixed